templates/1-basic_workflow_with_config/config1.yml

```diff
@@ -1,14 +1,43 @@
-# example of a config for the workflow
+# Example of a config file for the workflow
```

Comment:
@juliettelavoie
I added quite some detail here.
My suggestion would be to comment the first task similarly detailed. After that, subsequent tasks all follow a similar scheme of iteration loops, anyway.